src/cloud-api-adaptor/libvirt/kcli_cluster.sh

```diff
@@ -120,7 +122,8 @@ usage () {
 	          CLUSTER_VERSION         (default "${CLUSTER_VERSION}")
 	          LIBVIRT_NETWORK         (default "${LIBVIRT_NETWORK}")
 	          LIBVIRT_POOL            (default "${LIBVIRT_POOL}")
-	          CLUSTER_WORKERS         (default "${CLUSTER_WORKERS}").
+	          CLUSTER_WORKERS         (default "${CLUSTER_WORKERS}")
+	          CONTAINER_RUNTIME       (default "${CONTAINER_RUNTIME}". Or set to "crio")
```

Comment:
Do you think it's important to mention crio here? It's not anywhere else (actually it's even odd to say default as it actually uses the current value but that is quite common; still it adds to obscurity of getting `CONTAINER_RUNTIME    (default "crio". Or set to "crio")`

Reply:
you are right. I just wanted to emphasize it's "crio" but not "cri-o" :)
I will fix it.